### PR TITLE
fix(ui-components): enable test coverage for ui-component's tsx files

### PR DESCRIPTION
**Pull request by 815are**
enable sonar test coverage for ui-component's tsx files

### CODE REVIEW

**Comment by 815are** [comment]
I am not sure, do we need generate changelog in such case?

**Comment by tobiasqueck** [comment]
> I am not sure, do we need generate changelog in such case?

Not needed here. A changelog is only required if you want to release a change.

File: packages/ui-components/.storybook/main.js [hunk 1]
```diff
@@ -41,8 +41,6 @@ module.exports = {
             include: path.resolve(__dirname, '../')
         });
         config.resolve.extensions.push('.ts', '.tsx');
-        // Temporary fix, while root issue is not fixed in webpack 5 - https://github.com/webpack/webpack/issues/13691
```
Comment (devinea) [inline packages/ui-components/.storybook/main.js:44]: 👍

File: sonar-project.properties [hunk 2]
```diff
@@ -1,11 +1,11 @@
 sonar.organization=sap-1
 sonar.projectKey=SAP_open-ux-tools
 sonar.sources=.
-sonar.inclusions=**/*.ts
+sonar.inclusions=**/*.ts, packages/ui-components/src/**/*.tsx
```
Comment (devinea) [inline sonar-project.properties:4]:
```suggestion
sonar.inclusions=**/*.ts, **/*.tsx
```
Should this be more general?

**Review by devinea — COMMENTED** [review]
Thanks @815are for the fix. Few minor comments. 
Probably want a changeset for the changes in 
`UIMessageBar.tsx` and storybook?

**Comment by tobiasqueck** [comment]
> Probably want a changeset for the changes in
> `UIMessageBar.tsx` and storybook?

Ooops. Yes, please ignore my previous statement. @devinea is correct, you have changed something in `src` therefore, it needs a change set.

File: sonar-project.properties (hunk 2 above)
Comment (815are) [reply sonar-project.properties:4]: I can try, but I assume it would start to track coverage for `*.stories.tsx`

Comment (815are) [reply sonar-project.properties]: @devinea 
your proposed variant works as expected - commit 2cdafaa3726aca9ec7803f78e1b8755a4f7da3e2

**Review by Klaus-Keller — APPROVED** [review]
Thanks @815are, code looks now cleaner.
- code looks good
- test coverage is great
- did not build and test locally

Approved.

**Review by devinea — APPROVED** [review]

File: sonar-project.properties [hunk 3]
```diff
@@ -1,11 +1,11 @@
 sonar.organization=sap-1
 sonar.projectKey=SAP_open-ux-tools
 sonar.sources=.
-sonar.inclusions=**/*.ts
+sonar.inclusions=**/*.ts, **/*.tsx
 sonar.exclusions=**/*.test.ts,**/templates/**, **/examples/**/*.ts, packages/fe-fpm-writer/test/**/*.ts, packages/store/test/**/*.ts
```
[the remark this answers is not in the document]
Comment (815are) [reply sonar-project.properties]: thanks, updated exclusions with commit 463c8e2deafc6dfcd12943e6acd8ea0c7f42d29b

**Review by devinea — APPROVED** [review]
Thanks @815are 
Changes look good.

**Review by Klaus-Keller — APPROVED** [review]
I've checked the updated sonar exclusions, looks good.
Approved again.

**Pull request merged**